prepare/templates/summarization/general.py

```diff
@@ -0,0 +1,66 @@
+from src.unitxt.catalog import add_to_catalog
+from src.unitxt.templates import InputOutputTemplate, TemplatesList
+
+add_to_catalog(
+    InputOutputTemplate(
+        input_format="Produce a succinct summary for the following text, extracting the fundamental concepts and "
+        "crucial information.\n Text: {text}",
+        output_format="{target}",
+    ),
+    "templates.summarization.general.formal",
```

Comment:
i guess here too that the template names should match https://github.com/IBM/unitxt/tree/main/prepare/tasks/summerization ? though there is only one task there `tasks.summarization.abstractive`.